doc/man/scons.xml

```diff
@@ -314,15 +319,33 @@ should be built (on Windows systems):</para>
 scons C:\ D:\
 </literallayout>
 
-<para>To build only specific targets,
-supply them as command-line arguments:</para>
+<para>A subset of a hierarchical tree may be built by
```

Comment:
Note this was moved up from lower down for logical flow reasons, this change is not new content.